cmd/revendor/run-glide.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/Masterminds/glide/action"
+	"github.com/Masterminds/glide/repo"
+	"github.com/mongodb/grip"
+	"github.com/pkg/errors"
+)
+
+func main() {
+	var (
+		pkg      string
+		revision string
+	)
+	flag.StringVar(&pkg, "package", "", "name of the vendored package to update")
+	flag.StringVar(&revision, "revision", "", "revision to which to update")
+	flag.Parse()
+	if pkg == "" || revision == "" {
+		grip.Info("no repo to revendor, run-glide will no-op")
+		return
+	}
+
+	updateGlide(pkg, revision)
+
+	installer := repo.NewInstaller()
+	action.EnsureGoVendor()
+	action.Install(installer, false, false)
+}
+
+func updateGlide(pkg, revision string) {
+	wd, err := os.Getwd()
+	grip.EmergencyFatal(errors.Wrap(err, "error getting working dir"))
+	vendorPath := filepath.Join(wd, "vendor", pkg)
+	_, err = os.Stat(vendorPath)
+	grip.EmergencyFatal(errors.Wrapf(err, "vendor directory %s not found", vendorPath))
```

Comment:
nit: to increase readability, put a blank line after the fatal. 

real: check os.IsNotExist and stat.IsDir() in this situation, the error value from stat is weird.

Reply:
done